samples/sample13/Ballerina.toml

```diff
@@ -2,3 +2,10 @@
 org-name = "kathy"
 version = "0.0.1"
 
+[platform]
+target = "java8"
+
+[[platform.libraries]]
+artifactId = "toml4j"
+  path = "./libs/mysql-connector-java-8.0.11.jar"
+  module = "toml4j"
```

Comment:
Indentation is wrong. Refer https://github.com/ballerina-platform/ballerina-lang/blob/master/cli/ballerina-packerina/src/test/resources/test-resources/valid-project/Ballerina.toml

Reply:
Add the 'version' and 'groupId' as well. Refer to nexus maven for the correct values..

Reply:
Indented.

Reply:
Updated the .toml with the values